<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ANHROG" ignored="0" name="Selenium Test 1" tests="1" failures="1" timestamp="23 Aug 2017 04:48:07 GMT" time="12.793" errors="0">
  <testcase name="selenium_testmethod1" time="2.307" classname="testclasses.SeleniumTest1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at testclasses.SeleniumTest1.selenium_testmethod1(SeleniumTest1.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- selenium_testmethod1 -->
</testsuite> <!-- Selenium Test 1 -->
